lambdas/authorizationcode/src/test/java/uk/gov/di/ipv/cri/passport/authorizationcode/validation/AuthRequestValidatorTest.java

```diff
@@ -63,7 +63,7 @@ void validateRequestReturnsErrorResponseForNullParams() {
 
     @Test
     void validateRequestReturnsErrorResponseForEmptyParameters() {
-        var validationResult = validator.validateRequest(Collections.emptyMap());
+        var validationResult = validator.validateRequest(Collections.emptyMap(), "test-user-id");
```

Comment:
Is it worth shifting "test-user-id" to a constant if it's being reused?

Reply:
Yeah good shout, I'll change it